From 0ee0b79a5c923f58c11061d10ed538102d22e0df Mon Sep 17 00:00:00 2001
From: Silvio Traversaro <silvio@traversaro.it>
Date: Sat, 23 Nov 2024 11:08:12 +0100
Subject: [PATCH] Disable setting different flags in arm and ppc

---
 support/cmake/setArchitecture.cmake | 22 ----------------------
 1 file changed, 22 deletions(-)

diff --git a/support/cmake/setArchitecture.cmake b/support/cmake/setArchitecture.cmake
index dd2904a..a846b5c 100644
--- a/support/cmake/setArchitecture.cmake
+++ b/support/cmake/setArchitecture.cmake
@@ -145,28 +145,6 @@ endfunction()
 function(getArchitectureFlags ARCH COMPILE_FLAGS LINK_FLAGS)
   # message("Setting projects for ${ARCH} bits build")
   targetarchitecture(CPUARCH)
-  if(NOT ((ARCH EQUAL 32) OR (ARCH EQUAL 64)))
-    message(ERROR_CRITICAL "Please define the architecture")
-  endif()
-  if(UNIX AND NOT CPUARCH MATCHES "arm")
-    if(CPUARCH MATCHES "ppc64") # on AIX
-      if(ARCH EQUAL 32)
-        set(CCFLAG "")
-      elseif(ARCH EQUAL 64)
-        set(CCFLAG "-maix64")
-      endif()
-    else()
-      if(ARCH EQUAL 32)
-        set(CCFLAG "-m32")
-      elseif(ARCH EQUAL 64)
-        set(CCFLAG "-m64")
-      endif()
-    endif()
-    set(${COMPILE_FLAGS}
-        ${CCFLAG} PARENT_SCOPE)
-    set(${LINK_FLAGS}
-        ${CCFLAG} PARENT_SCOPE)
-  endif()
 endfunction()
 
 function(setArchitectureFlags ARCH)
